File: pset6/mario/more/more.py
from cs50 import get_int
# Loop for taking input from user
while True:
    h = get_int("Height: ")
    if h < 9 and h > 0:
        break

# for loop of printing pattern 
for i in range(1, h + 1):
    print(" " * (h - i) + "#" * i + "  " + ("#" * i))

File: pset1/mario/more/more.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;   //Declaration of number of columns of pyramid
    
    //To check input is between 1 and 8
    do
    {
        n  = get_int("Height: ");
    }
    while ((n < 1) || (n > 8));

    //Making pyramid
    for (int i = 0; i < n; i++)
    {
        for (int j = n; j > i + 1; j--) //Inserting spaces
        {   
            printf(" ");
        }
        for (int j = 0; j <= i; j++) //Making righ-aligned traingle
        {
            printf("#"); //Space between two pyramids
        }
        
        printf("  ");
        
        for (int j = 0; j <= i; j++) //Making lef-aligned triangle
        {
            printf("#");
        }
        
        printf("\n");
                
    }
    
}